x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/view/ViewRestTests.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.esql.view;
+
+import org.elasticsearch.core.TimeValue;
+
+import static org.elasticsearch.xpack.esql.plugin.EsqlFeatures.ESQL_VIEWS_FEATURE_FLAG;
+
+public class ViewRestTests extends AbstractViewTestCase {
```

Comment:
Are all the changes in `‎x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/view` only for this test class? Or are you planning on adding more tests in the future? The amount of plumbing/infra we do for this test is excessive and unnecessary; that's why I'm asking.